src/components/MessagesList/MessagesGroup/Message/Message.vue

```diff
@@ -59,6 +59,16 @@ the main body of the message as well as a quote.
 			<div class="message__main__right">
 				<div v-if="isTemporary && !isTemporaryUpload" class="icon-loading-small" />
 				<span v-if="hasDate" v-tooltip.auto="messageDate">{{ messageTime }}</span>
+				<CheckAll
```

Comment:
would be good to add a v-tooltip and also an aria-label for these icons

Reply:
https://github.com/nextcloud/spreed/pull/4231#issuecomment-741777019

> UI will be polished by @ma12-co in a follow up.

We should get the API in asap, so mobile apps can start their development and maybe finish before xmas